.github/workflows/runAqa.yml

```diff
@@ -0,0 +1,193 @@
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: comment_body
+          })
+    - name: Get workflow run info
+      run: |
+        echo ::set-output name=url::$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
+        echo ::set-output name=id::$GITHUB_RUN_ID
+      id: workflow_run_info
+    - name: Create success comment
+      uses: actions/github-script@v3
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          comment_body = `
+          @${{ github.actor }} Build(s) started with the following parameters:
+          - sdk_resource: ${{ steps.argparse.outputs.sdk_resource }}
+          - build_list: ${{ steps.argparse.outputs.build_list }}
+          - target: ${{ steps.argparse.outputs.target }}
+          - platform: ${{ steps.argparse.outputs.platform }}
+          - jdk_version: ${{ steps.argparse.outputs.jdk_version }}
+          - jdk_impl: ${{ steps.argparse.outputs.jdk_impl }}
+
+          Workflow Run ID: [${{ steps.workflow_run_info.outputs.id }}](${{ steps.workflow_run_info.outputs.url }})
+          `;
+          github.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: comment_body
+          })
+    - name: Echo parameters
+      run: |
+        echo build_parameters: ${{ steps.argparse.outputs.build_parameters }}
+        echo sdk_resource: ${{ steps.argparse.outputs.sdk_resource }}
+        echo build_list: ${{ steps.argparse.outputs.build_list }}
+        echo target: ${{ steps.argparse.outputs.target }}
+        echo platform: ${{ steps.argparse.outputs.platform }}
+        echo jdk_version: ${{ steps.argparse.outputs.jdk_version }}
+        echo jdk_impl: ${{ steps.argparse.outputs.jdk_impl }}
+
+  runBuild:
+    runs-on: ${{ matrix.platform }}
+    needs: parseComment
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.parseComment.outputs.build_parameters) }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: AdoptOpenJDK/install-jdk@v1
+      with:
+        version: ${{ matrix.jdk_version }}
+        source: ${{ matrix.sdk_resource }}
+        impl: ${{ matrix.jdk_impl }}
+    # get-pr step by @Simran-B https://github.com/actions/checkout/issues/331#issuecomment-707103442
+    - uses: actions/github-script@v3
+      id: get-pr
+      with:
+        script: |
+          const request = {
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            pull_number: context.issue.number
+          }
+          core.info(`Getting PR #${request.pull_number} from ${request.owner}/${request.repo}`)
+          try {
+            const result = await github.pulls.get(request)
+            return result.data
+          } catch (err) {
+            core.setFailed(`Request failed with error ${err}`)
+          }
+    - name: Extract head repo and branch
```

Comment:
As repo and branch are only used once in AQA step, I would suggest remove this echo step. AQA can use `${{ fromJSON(steps.get-pr.outputs.result).head.repo.full_name }}` and `${{ fromJSON(steps.get-pr.outputs.result).head.ref }}` directly.

Reply:
I have that initially but it refused to work for some reason. I can try again though.

Reply:
Leaving the echo step in would also let a person check that the correct repo and branch were checked out in the workflow job logs.

Reply:
Builds appear to run successfully after removing the echo step and using the variables directly.
However there isn't a way to tell which repo and branch it checked out in the logs, as far as I can tell.

Reply:
This actually same issue with https://github.com/AdoptOpenJDK/openjdk-tests/pull/2222#discussion_r577228967. run-aqa used to check if openjdk-tests is checkout, if yes, will ignore the parameter 'openjdk_testRepo'. Updated run-aqa with the check removed. The console output would have something like `/usr/bin/git clone --depth 1 -b branch https://github.com/user/openjdk-tests.git`.